.github/workflows/integration.yml

```diff
@@ -0,0 +1,86 @@
+name: Integration Tests
+
+on:
+  pull_request:
+    branches-ignore:
+      - 'ubuntu/**'
+
+concurrency:
+  group: 'ci-${{ github.workflow }}-${{ github.ref }}'
+  cancel-in-progress: true
+
+defaults:
+  run:
+    shell: sh -ex {0}
+
+env:
+  RELEASE: bionic
+
+jobs:
+  package-build:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: "Checkout #1"
+        uses: actions/checkout@v3.0.0
+      - name: "Checkout #2 (for tools/read-version)"
+        run: |
+          git fetch --unshallow
+          git remote add upstream https://git.launchpad.net/cloud-init
```

Comment:
Do we actually need to add LP as a remote, especially given that it is added but never fetched?

Reply:
That was probably a relic / copy-paste, I have removed it and unified Checkout #1 and #2 in a single one.

Reply:
Nice and TIL about `fetch-depth`, thanks!